README.md

```diff
@@ -1,26 +1,30 @@
 # Mount React Component
 
-This component was born from the need to be able to mount react components inside HTML views using Ruby on Rails, providing a simple way to render the react component on the view passing different parameters and recieving them on the component through `props`
+This component was born from the need to be able to mount react components inside HTML views using Ruby on Rails (although it should be able to be used in any other web-framework), providing a simple way to render the react component on the view passing different parameters and recieving them on the component through `props`.
 ## Usage
 
-Define an element to mount the component into in your server-side rendered html, specifying either an `#id` or a `.class`. You can use the `data-props` attribute to send props coming out directly of your server-side rendered html.
+Define an element to mount the component into in your server-side rendered html, specifying the component name on `data-react-component`. You can use the `data-react-props` attribute to send props coming out directly of your server-side rendered html.
 
 ```erb
 <div
-  id="my-react-component"
-  data-props="<%={
+  data-react-component="MyReactComponent"
+  data-react-props="<%={
```

Comment:
I'd reconsider using `react` as part of the attribute name. There are several reasons:

1. it's cumbersome to add `react-` to every component I want to use. (more typing)
2. this node module is intended to use if and only if you're using react, so it's a bit redundant
3. both `data-props` and `data-component` don't collide with any other attribute used in any other framework, so they'd be safe to use